statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -767,14 +767,35 @@ class VARProcess(object):
     -------
     **Attributes**:
     """
-    def __init__(self, coefs, exog, sigma_u, names=None):
+    def __init__(self, coefs, coefs_exog, sigma_u, names=None, _params_info=None):
```

Comment:
is `_params_info` the best name for an argument?  private arg names seem kinda weird

Reply:
VARProcess is mostly an internal class.
private `_params_info` is weird but the best I managed to come up with for this not so fast quickfix.
(It might become public, official but only after seeing how it works for a while. I still don't like several parts in the separation and duplication between process and results class.)